reporter/src/main/kotlin/model/EvaluatedModel.kt

```diff
@@ -138,8 +138,8 @@ data class EvaluatedModel(
             propertyNamingStrategy = PROPERTY_NAMING_STRATEGY
         }
 
-        private val JSON_MAPPER = JsonMapper().apply(MAPPER_CONFIG)
-        private val YAML_MAPPER = YAMLMapper().apply(MAPPER_CONFIG)
+        private val JSON_MAPPER by lazy { JsonMapper().apply(MAPPER_CONFIG) }
+        private val YAML_MAPPER by lazy { YAMLMapper().apply(MAPPER_CONFIG) }
```

Comment:
Nit: "only create them if they are required" is basically just the definition of lazy initialization. To me the questions are

- Is it expensive to create these mappers?
- Is it likely that they are *not* required?

Only if both questions can be answered with "yes" (and the according rationale is given in the commit message), I'd say it makes sense to use lazy initialization.